packages/neuron-ui/src/components/NervosDAO/index.tsx

```diff
@@ -56,27 +57,34 @@ const NervosDAO = ({
       }
       clearTimeout(timer)
       timer = setTimeout(() => {
+        setErrorMessage('')
+        clearGeneratedTx()
+
+        const verifyRes = verifyAmount(value)
+        if (verifyRes !== true) {
+          setErrorMessage(t(`messages.codes.${verifyRes.code}`, { fieldName: 'deposit' }))
+          return
+        }
+
         if (+value < MIN_DEPOSIT_AMOUNT) {
```

Comment:
Please using BigInt for `value` and `MIN_DEPOSIT_AMOUNT`

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/1093/commits/f96c1543d5f881a2822931f8f27291c48dfd0312